src/components/accordion/Accordion.tsx

```diff
@@ -1,11 +1,15 @@
-import { FC } from "react";
+import { FC, useMemo } from "react";
 import { Accordion as BaseAccordion, AccordionProps as BaseAccordionProps } from "baseui/accordion";
 import { getAccordionOverrides } from "./overrides";
+import { getMergedOverrides } from "../../shared/utils/getMergedOverrides";
 
 export type AccordionProps = BaseAccordionProps;
 
-const Accordion: FC<AccordionProps> = ({ ...props }) => {
-  const overrides = getAccordionOverrides();
+const Accordion: FC<AccordionProps> = ({ overrides: baseOverrides, ...props }) => {
```

Comment:
Without memoizing overrides object on parent level this makes a little sense, because on every parent render  ```useMemo``` will trigger recalculation anyway (link to overrides object changes in that case)
So i would better omit using ```useMemo``` inside the component at all, and let developer  to play with wrapping this component inside ```memo``` on parent level, but only if it is needed.